src/openms/source/DATASTRUCTURES/SuffixArraySeqan.cpp

```diff
@@ -171,7 +171,7 @@ namespace OpenMS
   //destructor
   SuffixArraySeqan::~SuffixArraySeqan()
   {
-
+    delete it_;
```

Comment:
for minimal invasive surgery, we could just make it a proper object, not 
a pointer.
Then there is no need for delete.

Am 1/15/2015 um 1:15 PM schrieb Hannes Roest:
>> @@ -171,7 +171,7 @@ namespace OpenMS
>>     //destructor
>>     SuffixArraySeqan::~SuffixArraySeqan()
>>     {
>> -
>> +    delete it_;
>
> and document what is wrong with it...maybe we could call delete on the old object before creating a new one?
>
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/OpenMS/OpenMS/pull/1148/files#r23005841
>